### DISCUSSION TITLE
I created a theme no one will like, but it may help anyone with no experience create themes easily

### DISCUSSION THREAD

**Discussion by deminimis** [zim-desktop-wiki/zim-desktop-wiki#2453] [Show and tell]
I went through the css files of some themes and the gtk3 manual, and created a ~10kb theme that uses black backgrounds and pale yellow font, and removed most animations. 

I also wrote down what each css does to some extent. Under the extras folder is a full list of what I found that changes elements. There are others, but that seems to cover most elements. 

https://github.com/jonsnow231/minimal-resources-zim

**Comment by supertoothy** [comment]
Thank you.
